sema4ai/tests/sema4ai_code_tests/test_vscode_integration.py

```diff
@@ -2770,3 +2770,85 @@ def test_update_agent_version(
 
     assert not result["success"]
     assert result["message"] == "Version entry was not found in the agent spec."
+
+
+def test_text_document_code_actions(language_server_initialized) -> None:
+    from sema4ai_code import commands
+
+    ls_client = language_server_initialized
+
+    win_uri = "file:///x%3A/vscode-robot/local_test/robot_check/agent-spec.yaml"
```

Comment:
I think it may be better to use the `tmpdir` fixture and then `uris.to_fs_path(uri)` and `uris.from_fs_path(path)` as needed instead of checking OS-specific things here...